core/node/events/events.go

```diff
@@ -70,6 +70,33 @@ func MakeStreamEvent(
 	return event, nil
 }
 
+func MakeStreamEventWithTags(
+	wallet *crypto.Wallet,
+	payload IsStreamEvent_Payload,
+	prevMiniblockHash []byte,
```

Comment:
```suggestion
	prevMiniblock *MiniblockRef,
```
Event has PrevMiniblockNum now - it helps to detect cases when local node is behind to hand it correctly during event add. (see comment below)